.github/workflows/testacc.yml

```diff
@@ -90,6 +90,7 @@ jobs:
           ASTRO_API_HOST: https://api.astronomer-dev.io
           SKIP_CLUSTER_RESOURCE_TESTS: ${{ env.SKIP_CLUSTER_RESOURCE_TESTS }}
           HOSTED_TEAM_ID: clwbclrc100bl01ozjj5s4jmq
+          TF_HOSTED_TEAM_ID: clx44rvzr01nc01o06pze6qb7
```

Comment:
can we use the same team_id instead of adding a new env var

Reply:
i can remove these env vars for now - i'm still using the hosted team id ones